packages/devtools_app/lib/src/code_size/code_size_screen.dart

```diff
@@ -45,6 +45,8 @@ class CodeSizeScreen extends Screen {
   @visibleForTesting
   static const diffViewTreemapKey = Key('Diff View Treemap');
 
+  static const loadingMessage = 'Loading data...\nUI may be unresponsive.';
```

Comment:
nit: how about "Loading data...\nPlease do not refresh or leave this page."

Reply:
sg, done